gettext-full: build error (BISON_LOCALEDIR undefined)

### Describe the bug

Some of very recent commits broke the build `gettext-full`. I am currently at 6b981c54647ae951fcba01edc5fd27feb29e2e0d which is broken, while just few days ago at ac9a97e49b8cd46cd19cbfdeb147b3815f5af0ba was fine.



### OpenWrt version

master

### OpenWrt release

master

### OpenWrt target/subtarget

ramips/mt7621

### Device

NA

### Image kind

Self-built image

### Steps to reproduce

Building the image from current master fails at building `gettext-full` with error:

```log
WARNING: Makefile 'package/utils/audit/Makefile' has a dependency on 'libev', which does not exist
WARNING: Makefile 'package/utils/busybox/Makefile' has a dependency on 'libpam', which does not exist
WARNING: Makefile 'package/utils/busybox/Makefile' has a dependency on 'libtirpc', which does not exist
WARNING: Makefile 'package/utils/busybox/Makefile' has a dependency on 'libpam', which does not exist
WARNING: Makefile 'package/utils/busybox/Makefile' has a dependency on 'libtirpc', which does not exist
WARNING: Makefile 'package/utils/busybox/Makefile' has a build dependency on 'libpam', which does not exist
WARNING: Makefile 'package/network/services/lldpd/Makefile' has a dependency on 'libnetsnmp', which does not exist
WARNING: Makefile 'package/feeds/packages/perl/perl-device-usb/Makefile' has a dependency on 'libusb-compat', which does not exist
WARNING: Makefile 'package/utils/policycoreutils/Makefile' has a dependency on 'libpam', which does not exist
WARNING: Makefile 'package/utils/policycoreutils/Makefile' has a dependency on 'libpam', which does not exist
WARNING: Makefile 'package/utils/policycoreutils/Makefile' has a build dependency on 'libpam', which does not exist
make[2]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/scripts/config'
make[2]: 'conf' is up to date.
make[2]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/scripts/config'
make[1]: Entering directory '/home/risa/work/openwrt-build-env/openwrt'
make[2]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/package/devel/gperf'
make[2]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/package/devel/gperf'
time: package/devel/gperf/host-compile#0.09#0.01#0.10
make[2]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/package/libs/gnulib-l10n'
make[2]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/package/libs/gnulib-l10n'
time: package/libs/gnulib-l10n/host-compile#0.06#0.00#0.07
make[2]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/package/libs/libiconv-full'
make[2]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/package/libs/libiconv-full'
time: package/libs/libiconv-full/host-compile#0.08#0.01#0.09
make[2]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/package/libs/libunistring'
make[2]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/package/libs/libunistring'
time: package/libs/libunistring/host-compile#0.10#0.01#0.10
make[2]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/package/libs/libxml2'
make[2]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/package/libs/libxml2'
time: package/libs/libxml2/host-compile#0.08#0.01#0.08
make[2]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/package/libs/gettext-full'
printf '%s\n%s\n' 'all:' 'install:' > /home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gnulib-local/m4/Makefile;  printf '%s\n%s\n' 'all:' 'install:' > /home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/libtextstyle/gnulib-local/Makefile;  printf '%s\n%s\n' 'all:' 'install:' > /home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/libtextstyle/m4/Makefile;  printf '%s\n%s\n' 'all:' 'install:' > /home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/m4/Makefile; 
CFLAGS="-O2 -I/home/risa/work/openwrt-build-env/openwrt/staging_dir/host/include -I/home/risa/work/openwrt-build-env/openwrt/staging_dir/hostpkg/include -I/home/risa/work/openwrt-build-env/openwrt/staging_dir/target-mipsel_24kc_musl/host/include -DPIC -fPIC" CPPFLAGS="-I/home/risa/work/openwrt-build-env/openwrt/staging_dir/host/include -I/home/risa/work/openwrt-build-env/openwrt/staging_dir/hostpkg/include -I/home/risa/work/openwrt-build-env/openwrt/staging_dir/target-mipsel_24kc_musl/host/include" CXXFLAGS="-O2 -I/home/risa/work/openwrt-build-env/openwrt/staging_dir/host/include -I/home/risa/work/openwrt-build-env/openwrt/staging_dir/hostpkg/include -I/home/risa/work/openwrt-build-env/openwrt/staging_dir/target-mipsel_24kc_musl/host/include" LDFLAGS="-L/home/risa/work/openwrt-build-env/openwrt/staging_dir/host/lib -L/home/risa/work/openwrt-build-env/openwrt/staging_dir/hostpkg/lib -L/home/risa/work/openwrt-build-env/openwrt/staging_dir/target-mipsel_24kc_musl/host/lib" make -j1 -C /home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/.  SUBDIRS='$(foreach dir, intl intl-csharp intl-java its gnulib-lib gnulib-local lib $(SUBDIR_libasprintf) libgettextpo libgrep libtextstyle m4 misc po projects src styles gettext-runtime gettext-tools ,$(wildcard $(dir)) )'
make[3]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1'
make  all-recursive
make[4]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1'
Making all in gnulib-local
make[5]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gnulib-local'
make[5]: Nothing to be done for 'all'.
make[5]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gnulib-local'
Making all in libtextstyle
make[5]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/libtextstyle'
make  all-recursive
make[6]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/libtextstyle'
Making all in gnulib-local
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/libtextstyle/gnulib-local'
make[7]: Nothing to be done for 'all'.
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/libtextstyle/gnulib-local'
Making all in lib
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/libtextstyle/lib'
make  all-am
make[8]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/libtextstyle/lib'
make[8]: Nothing to be done for 'all-am'.
make[8]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/libtextstyle/lib'
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/libtextstyle/lib'
Making all in m4
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/libtextstyle/m4'
make[7]: Nothing to be done for 'all'.
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/libtextstyle/m4'
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/libtextstyle'
make[7]: Nothing to be done for 'all-am'.
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/libtextstyle'
make[6]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/libtextstyle'
make[5]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/libtextstyle'
Making all in m4
make[5]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/m4'
make[5]: Nothing to be done for 'all'.
make[5]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/m4'
Making all in gettext-runtime
make[5]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime'
make  all-recursive
make[6]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime'
Making all in intl
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/intl'
make  all-recursive
make[8]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/intl'
Making all in gnulib-lib
make[9]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/intl/gnulib-lib'
make  all-recursive
make[10]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/intl/gnulib-lib'
make[11]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/intl/gnulib-lib'
make[11]: Nothing to be done for 'all-am'.
make[11]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/intl/gnulib-lib'
make[10]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/intl/gnulib-lib'
make[9]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/intl/gnulib-lib'
make[9]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/intl'
make[9]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/intl'
make[8]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/intl'
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/intl'
Making all in intl-csharp
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/intl-csharp'
make[7]: Nothing to be done for 'all'.
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/intl-csharp'
Making all in intl-java
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/intl-java'
cd . && test ! -d /usr/lib/jdk1.3.1 || env PATH=/usr/lib/jdk1.3.1/bin:$PATH javadoc -d javadoc2 gnu.gettext gnu/gettext/*.java
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/intl-java'
Making all in gnulib-lib
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/gnulib-lib'
make  all-recursive
make[8]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/gnulib-lib'
make[9]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/gnulib-lib'
make[9]: Nothing to be done for 'all-am'.
make[9]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/gnulib-lib'
make[8]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/gnulib-lib'
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/gnulib-lib'
Making all in m4
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/m4'
make[7]: Nothing to be done for 'all'.
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/m4'
Making all in po
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/po'
Makefile:596: warning: overriding recipe for target 'en@quot.po-create'
Rules-quot:16: warning: ignoring old recipe for target 'en@quot.po-create'
Makefile:598: warning: overriding recipe for target 'en@boldquot.po-create'
Rules-quot:18: warning: ignoring old recipe for target 'en@boldquot.po-create'
Makefile:604: warning: overriding recipe for target '.insert-header.po-update-en'
Rules-quot:24: warning: ignoring old recipe for target '.insert-header.po-update-en'
Makefile:639: warning: overriding recipe for target 'en@quot.insert-header'
Rules-quot:59: warning: ignoring old recipe for target 'en@quot.insert-header'
Makefile:642: warning: overriding recipe for target 'en@boldquot.insert-header'
Rules-quot:62: warning: ignoring old recipe for target 'en@boldquot.insert-header'
Makefile:646: warning: overriding recipe for target 'mostlyclean-quot'
Rules-quot:66: warning: ignoring old recipe for target 'mostlyclean-quot'
make[7]: Nothing to be done for 'all'.
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/po'
Making all in src
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/src'
make[7]: Nothing to be done for 'all'.
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime/src'
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime'
make[7]: Nothing to be done for 'all-am'.
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime'
make[6]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime'
make[5]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-runtime'
Making all in gettext-tools
make[5]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools'
make  all-recursive
make[6]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools'
Making all in its
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/its'
make[7]: Nothing to be done for 'all'.
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/its'
Making all in gnulib-lib
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/gnulib-lib'
make  all-am
make[8]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/gnulib-lib'
make[8]: Nothing to be done for 'all-am'.
make[8]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/gnulib-lib'
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/gnulib-lib'
Making all in libgettextpo
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/libgettextpo'
make  all-am
make[8]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/libgettextpo'
make[8]: Nothing to be done for 'all-am'.
make[8]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/libgettextpo'
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/libgettextpo'
Making all in libgrep
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/libgrep'
make  all-am
make[8]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/libgrep'
make[8]: Nothing to be done for 'all-am'.
make[8]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/libgrep'
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/libgrep'
Making all in m4
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/m4'
make[7]: Nothing to be done for 'all'.
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/m4'
Making all in misc
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/misc'
make[7]: Nothing to be done for 'all'.
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/misc'
Making all in po
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/po'
Makefile:596: warning: overriding recipe for target 'en@quot.po-create'
Rules-quot:16: warning: ignoring old recipe for target 'en@quot.po-create'
Makefile:598: warning: overriding recipe for target 'en@boldquot.po-create'
Rules-quot:18: warning: ignoring old recipe for target 'en@boldquot.po-create'
Makefile:604: warning: overriding recipe for target '.insert-header.po-update-en'
Rules-quot:24: warning: ignoring old recipe for target '.insert-header.po-update-en'
Makefile:639: warning: overriding recipe for target 'en@quot.insert-header'
Rules-quot:59: warning: ignoring old recipe for target 'en@quot.insert-header'
Makefile:642: warning: overriding recipe for target 'en@boldquot.insert-header'
Rules-quot:62: warning: ignoring old recipe for target 'en@boldquot.insert-header'
Makefile:646: warning: overriding recipe for target 'mostlyclean-quot'
Rules-quot:66: warning: ignoring old recipe for target 'mostlyclean-quot'
make[7]: Nothing to be done for 'all'.
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/po'
Making all in projects
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/projects'
make[7]: Nothing to be done for 'all'.
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/projects'
Making all in src
make[7]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/src'
make  all-am
make[8]: Entering directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/src'
/home/risa/work/openwrt-build-env/openwrt/staging_dir/host/bin/gcc -DLOCALEDIR=\"/home/risa/work/openwrt-build-env/openwrt/staging_dir/hostpkg/share/locale\" -DUSEJAVA=0 -DGETTEXTJAR=\"/home/risa/work/openwrt-build-env/openwrt/staging_dir/hostpkg/share/gettext\"\"/gettext.jar\" -DBINDIR=\"/home/risa/work/openwrt-build-env/openwrt/staging_dir/hostpkg/bin\" -DLIBDIR=\"/home/risa/work/openwrt-build-env/openwrt/staging_dir/hostpkg/lib\" -DLIBEXECDIR=\"/home/risa/work/openwrt-build-env/openwrt/staging_dir/hostpkg/libexec\" -DGETTEXTDATADIR=\"/home/risa/work/openwrt-build-env/openwrt/staging_dir/hostpkg/share/gettext\" -DPROJECTSDIR=\"/home/risa/work/openwrt-build-env/openwrt/staging_dir/hostpkg/share/gettext\"\"/projects\" -DEXEEXT=\"\" -DHAVE_CONFIG_H -I. -I..  -I.. -I../tree-sitter-0.23.2/lib/include -I../libgrep -I../libgrep -I../gnulib-lib -I../gnulib-lib -I../../gettext-runtime/intl -I../../gettext-runtime/intl -DINSTALLDIR=\"/home/risa/work/openwrt-build-env/openwrt/staging_dir/hostpkg/bin\" -I/home/risa/work/openwrt-build-env/openwrt/staging_dir/host/include -I/home/risa/work/openwrt-build-env/openwrt/staging_dir/hostpkg/include -I/home/risa/work/openwrt-build-env/openwrt/staging_dir/target-mipsel_24kc_musl/host/include  -O2 -I/home/risa/work/openwrt-build-env/openwrt/staging_dir/host/include -I/home/risa/work/openwrt-build-env/openwrt/staging_dir/hostpkg/include -I/home/risa/work/openwrt-build-env/openwrt/staging_dir/target-mipsel_24kc_musl/host/include -DPIC -fPIC -c -o msgcmp-msgcmp.o `test -f 'msgcmp.c' || echo './'`msgcmp.c
In file included from msgcmp.c:35:
msgcmp.c: In function 'main':
msgcmp.c:110:46: error: 'BISON_LOCALEDIR' undeclared (first use in this function)
  110 |   bindtextdomain ("bison-runtime", relocate (BISON_LOCALEDIR));
      |                                              ^~~~~~~~~~~~~~~
../gnulib-lib/relocatable.h:114:29: note: in definition of macro 'relocate'
  114 | #define relocate(pathname) (pathname)
      |                             ^~~~~~~~
msgcmp.c:110:46: note: each undeclared identifier is reported only once for each function it appears in
  110 |   bindtextdomain ("bison-runtime", relocate (BISON_LOCALEDIR));
      |                                              ^~~~~~~~~~~~~~~
../gnulib-lib/relocatable.h:114:29: note: in definition of macro 'relocate'
  114 | #define relocate(pathname) (pathname)
      |                             ^~~~~~~~
make[8]: *** [Makefile:5539: msgcmp-msgcmp.o] Error 1
make[8]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/src'
make[7]: *** [Makefile:4374: all] Error 2
make[7]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools/src'
make[6]: *** [Makefile:3591: all-recursive] Error 1
make[6]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools'
make[5]: *** [Makefile:3434: all] Error 2
make[5]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/gettext-tools'
make[4]: *** [Makefile:411: all-recursive] Error 1
make[4]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1'
make[3]: *** [Makefile:367: all] Error 2
make[3]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1'
make[2]: *** [Makefile:188: /home/risa/work/openwrt-build-env/openwrt/build_dir/hostpkg/gettext-0.24.1/.built] Error 2
make[2]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt/package/libs/gettext-full'
time: package/libs/gettext-full/host-compile#0.19#0.02#0.21
    ERROR: package/libs/gettext-full [host] failed to build.
make[1]: *** [package/Makefile:181: package/libs/gettext-full/host/compile] Error 1
make[1]: Leaving directory '/home/risa/work/openwrt-build-env/openwrt'
make: *** [/home/risa/work/openwrt-build-env/openwrt/include/toplevel.mk:233: package/libs/gettext-full/compile] Error 2
```

### Actual behaviour

_No response_

### Expected behaviour

_No response_

### Additional info

_No response_

### Diffconfig

```text
CONFIG_TARGET_ramips=y
CONFIG_TARGET_ramips_mt7621=y
CONFIG_TARGET_ramips_mt7621_DEVICE_tplink_archer-ax23-v1=y
CONFIG_DEVEL=y
CONFIG_TOOLCHAINOPTS=y
CONFIG_BUSYBOX_CUSTOM=y
# CONFIG_BINUTILS_USE_VERSION_2_42 is not set
CONFIG_BINUTILS_USE_VERSION_2_44=y
CONFIG_BINUTILS_VERSION="2.44"
CONFIG_BINUTILS_VERSION_2_44=y
# CONFIG_BPF_TOOLCHAIN_BUILD_LLVM is not set
CONFIG_BPF_TOOLCHAIN_NONE=y
CONFIG_BUSYBOX_CONFIG_DIFF=y
CONFIG_BUSYBOX_CONFIG_FEATURE_DEFAULT_PASSWD_ALGO="md5"
CONFIG_BUSYBOX_CONFIG_FEATURE_DIFF_DIR=y
CONFIG_BUSYBOX_CONFIG_FEATURE_DIFF_LONG_OPTIONS=y
# CONFIG_BUSYBOX_CONFIG_FEATURE_EDITING_SAVEHISTORY is not set
CONFIG_BUSYBOX_CONFIG_FEATURE_NSLOOKUP_LONG_OPTIONS=y
# CONFIG_BUSYBOX_CONFIG_NTPD is not set
CONFIG_BUSYBOX_CONFIG_PATCH=y
# CONFIG_BUSYBOX_CONFIG_SETSID is not set
# CONFIG_GCC_USE_VERSION_14 is not set
CONFIG_GCC_USE_VERSION_15=y
CONFIG_GCC_VERSION="15.1.0"
CONFIG_GCC_VERSION_15=y
# CONFIG_GDB is not set
CONFIG_GNUTLS_ALPN=y
CONFIG_GNUTLS_ANON=y
CONFIG_GNUTLS_DTLS_SRTP=y
CONFIG_GNUTLS_HEARTBEAT=y
CONFIG_GNUTLS_OCSP=y
CONFIG_GNUTLS_PSK=y
CONFIG_IB=y
CONFIG_IB_STANDALONE=y
# CONFIG_KERNEL_WERROR is not set
CONFIG_PACKAGE_atop=m
CONFIG_PACKAGE_chrony=m
CONFIG_PACKAGE_chrony-nts=m
CONFIG_PACKAGE_ddns-scripts=m
CONFIG_PACKAGE_ddns-scripts-services=m
CONFIG_PACKAGE_dnsmasq=m
CONFIG_PACKAGE_glib2=m
CONFIG_PACKAGE_hostapd-basic-mbedtls=m
CONFIG_PACKAGE_hostapd-utils=m
CONFIG_PACKAGE_htop=m
CONFIG_PACKAGE_iftop=m
CONFIG_PACKAGE_ip-bridge=m
CONFIG_PACKAGE_ip-full=m
CONFIG_PACKAGE_iperf3=m
CONFIG_PACKAGE_ipset=m
CONFIG_PACKAGE_iwcap=m
CONFIG_PACKAGE_kmod-crypto-kpp=m
CONFIG_PACKAGE_kmod-crypto-lib-chacha20=m
CONFIG_PACKAGE_kmod-crypto-lib-chacha20poly1305=m
CONFIG_PACKAGE_kmod-crypto-lib-curve25519=m
CONFIG_PACKAGE_kmod-crypto-lib-poly1305=m
CONFIG_PACKAGE_kmod-fs-exfat=m
CONFIG_PACKAGE_kmod-fs-vfat=m
CONFIG_PACKAGE_kmod-ipt-core=m
CONFIG_PACKAGE_kmod-ipt-ipset=m
CONFIG_PACKAGE_kmod-mii=m
CONFIG_PACKAGE_kmod-nf-ipt=m
CONFIG_PACKAGE_kmod-nls-base=y
CONFIG_PACKAGE_kmod-nls-cp437=m
CONFIG_PACKAGE_kmod-nls-iso8859-1=m
CONFIG_PACKAGE_kmod-nls-utf8=m
CONFIG_PACKAGE_kmod-ppp=m
CONFIG_PACKAGE_kmod-pppoe=m
CONFIG_PACKAGE_kmod-pppox=m
CONFIG_PACKAGE_kmod-scsi-core=m
CONFIG_PACKAGE_kmod-slhc=m
CONFIG_PACKAGE_kmod-udptunnel4=m
CONFIG_PACKAGE_kmod-udptunnel6=m
CONFIG_PACKAGE_kmod-usb-core=y
CONFIG_PACKAGE_kmod-usb-ehci=y
CONFIG_PACKAGE_kmod-usb-ledtrig-usbport=y
CONFIG_PACKAGE_kmod-usb-net=m
CONFIG_PACKAGE_kmod-usb-net-cdc-ether=m
CONFIG_PACKAGE_kmod-usb-net-rndis=m
CONFIG_PACKAGE_kmod-usb-ohci=m
CONFIG_PACKAGE_kmod-usb-printer=m
CONFIG_PACKAGE_kmod-usb-storage=m
CONFIG_PACKAGE_kmod-usb-uhci=m
CONFIG_PACKAGE_kmod-usb2=y
CONFIG_PACKAGE_kmod-wireguard=m
CONFIG_PACKAGE_libatomic=m
CONFIG_PACKAGE_libattr=m
CONFIG_PACKAGE_libbpf=m
CONFIG_PACKAGE_libcap=m
CONFIG_PACKAGE_libelf=m
CONFIG_PACKAGE_libevent2-core=m
CONFIG_PACKAGE_libevent2-pthreads=m
CONFIG_PACKAGE_libffi=m
CONFIG_PACKAGE_libgmp=m
CONFIG_PACKAGE_libgnutls=m
CONFIG_PACKAGE_libiperf3=m
CONFIG_PACKAGE_libipset=m
CONFIG_PACKAGE_libncurses=m
CONFIG_PACKAGE_libnettle=m
CONFIG_PACKAGE_libpcap=m
CONFIG_PACKAGE_libpcre2=m
CONFIG_PACKAGE_libpthread=y
CONFIG_PACKAGE_libstdcpp=m
CONFIG_PACKAGE_libustream-mbedtls=m
CONFIG_PACKAGE_nstat=m
CONFIG_PACKAGE_odhcp6c=m
CONFIG_PACKAGE_odhcpd-ipv6only=m
CONFIG_PACKAGE_opkg=y
# CONFIG_PACKAGE_ppp is not set
CONFIG_PACKAGE_pps-tools=m
CONFIG_PACKAGE_resolveip=m
CONFIG_PACKAGE_tcpdump=m
CONFIG_PACKAGE_terminfo=m
CONFIG_PACKAGE_ucode-mod-uloop=m
CONFIG_PACKAGE_vim=m
CONFIG_PACKAGE_vim-full=m
CONFIG_PACKAGE_vim-fuller=m
CONFIG_PACKAGE_vim-runtime=m
CONFIG_PACKAGE_wireguard-tools=m
# CONFIG_PACKAGE_wpad-basic-mbedtls is not set
# CONFIG_HTOP_LMSENSORS is not set
CONFIG_PACKAGE_kmod-lib-crc-ccitt=y
```

### Terms

- [x] I am reporting an issue for OpenWrt, not an unsupported fork.

--- a/tools/bison/Makefile
+++ b/tools/bison/Makefile
@@ -21,13 +21,13 @@ include $(INCLUDE_DIR)/host-build.mk
 HOST_CONFIGURE_ARGS += --enable-threads=posix --disable-nls
 
 define Host/Install
-	$(call Host/Install/Default)
+	$(call Host/Compile/Default,install aclocal_DATA=) # Macro provided by gnulib
 	$(INSTALL_BIN) ./scripts/yacc $(STAGING_DIR_HOST)/bin/yacc
 endef
 
 define Host/Uninstall
 	rm -f $(STAGING_DIR_HOST)/bin/yacc
-	-$(call Host/Compile/Default,uninstall)
+	-$(call Host/Compile/Default,uninstall aclocal_DATA=) # Macro provided by gnulib
 endef
 
 $(eval $(call HostBuild))

